Find a functionally equivalent code:android.widget.DatePicker.getMonth:COMMENT
Method Modifier: public      
Comment:/**
 * @return The selected month.
 */

@@ -1,3 +1,3 @@
 {
-    return mCurrentDate.get(Calendar.MONTH);
+    return mDelegate.getMonth();
 }

